aries-site/src/pages/templates/drill-down-navigation.mdx

```diff
@@ -1,3 +1,50 @@
-import { ComingSoon } from '../../components';
+import { DiagramPreview, Example } from '../../layouts';
+import { DrillDownCardExample, DrillDownDataTableExample, DrillDownListExample } from '../../examples';
 
-<ComingSoon />
\ No newline at end of file
+## When to Use:
+- When users need to focus on one task at a time.
+- To give information overview with the ability to drill down into specific information.
+- When screen real estate is limited.
+- When designing for many levels of functionality.
+- To divide meaningful content into sections.
+
+## Guidelines:
+- When the user is moving forward step-by-step to access further information.
```

Comment:
I feel like the first 3 bullet points (When the user.., When going from..., To allow users...) on this section would fit better in "When to use" since they aren't really actionable pieces of guidance.